### PR TITLE
Do not incomplete on connection error with ssh based consoles

### PR DESCRIPTION
**Pull request by Soulofdestiny**
[Verification run, bad case](http://opeth.suse.de/tests/3667#step/bootloader_s390/33)
[Verification run, good case](http://opeth.suse.de/tests/3668)

### CODE REVIEW

File: consoles/network_console.pm [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2019 SUSE LLC
```
Comment (Martchus) [inline consoles/network_console.pm]: Looks like a new file. So `# Copyright © 2019 SUSE LLC` should be sufficient.

File: consoles/network_console.pm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        $self->connect_remote($self->{args});
+        return $self->SUPER::activate;
+    }
+    catch {
+        die $_ unless blessed $_ && $_->can('rethrow');
+        return {error => $_->error};
+    };
+}
+
+# to be overloaded
```
Comment (Martchus) [inline consoles/network_console.pm]: I suppose it should be over*written*.

Comment (Soulofdestiny) [reply consoles/network_console.pm]: copy paste fail - fixed

File: consoles/network_console.pm (hunk 1 above)
Comment (Soulofdestiny) [reply consoles/network_console.pm]: done

**Comment by okurz** [comment]
And now the coverage dropped ?!?

**Comment by Soulofdestiny** [comment]
> And now the coverage dropped ?!?

weird indeed... I pushed again since the year for the testfile was still wrong - let's see

**Pull request merged**